src/main/scala/com/databricks/labs/overwatch/pipeline/PostProcessor.scala

```diff
@@ -85,16 +85,46 @@ class PostProcessor(config: Config) extends PipelineTargets(config) {
     logger.log(Level.INFO, invalidTargetsMsg)
     logger.log(Level.INFO, msg)
     // TODO -- spark_events_bronze -- put in proper rules -- hot fix due to optimization issues
-    Helpers.parOptimize(validatedTargets.filterNot(_.name == "spark_events_bronze"), maxFileSizeMB = 128)
-    Helpers.parOptimize(validatedTargets.filter(_.name == "spark_events_bronze"), maxFileSizeMB = 32)
+    Helpers.parOptimize(validatedTargets.filterNot(_.name == "spark_events_bronze"), maxFileSizeMB = 128,includeVacuum = true)
+    Helpers.parOptimize(validatedTargets.filter(_.name == "spark_events_bronze"), maxFileSizeMB = 32,includeVacuum = true)
+  }
+
+
+  /**
+   * launch the optimize jobs in parallel
+   */
+  private def executeExternalOptimize(orgID: String): Unit = {
+    appendIntermediateTargets
+    val validatedTargets = validateExists
+    val invalidTargets = tablesToOptimize.map(_.tableLocation).diff(validatedTargets.map(_.tableLocation))
+    val invalidTargetsMsg = s"OPTIMIZE: Missing Targets: \n${invalidTargets.mkString("\n")}"
+    val msg = s"OPTIMIZE: Targets To Optimize: \n${validatedTargets.map(_.tableLocation).mkString("\n")}"
+
+    if (config.debugFlag) {
+      println(invalidTargetsMsg)
+      println(msg)
+    }
+    logger.log(Level.INFO, invalidTargetsMsg)
+    logger.log(Level.INFO, msg)
+    // TODO -- spark_events_bronze -- put in proper rules -- hot fix due to optimization issues
+    if (orgID == ""){
+      println(s"Running Optimization for ${config.databaseName}")
+      Helpers.parOptimize(validatedTargets.filterNot(_.name == "spark_events_bronze"), maxFileSizeMB = 128,true,orgID)
```

Comment:
nitpick: arguments are the same for both conditional paths, except the print statement, they could be extracted

Reply:
Done.